commitlint.config.js

```diff
@@ -17,6 +17,6 @@ module.exports = {
     'subject-case': [2, 'always', ['sentence-case']],
     'body-max-line-length': [1, 'always', '150'],
     'scope-empty': [2, 'never'],
-    'scope-enum': [2, 'always', [...getPackageNames(), 'repo', 'release']],
+    'scope-enum': [2, 'always', [...getPackageNames(), 'repo', 'release', '*']],
```

Comment:
If you want to write a commit message that affects everything inside `packages`, we should use `*`. So this just adds it as it was missing

Reply:
This was missing, indeed! Great job!